### PR TITLE
added-pdf-export-feature

### LINKED ISSUE #12: Export Resume as PDF
Once a user has added all their information to the resume, they are able to export as a PDF. PDF must not crop off any information for the resume, but not include UI elements from the web page.

### PR DESCRIPTION
**Pull request by sammychinedu2ky**
Closes #12

### CODE REVIEW

File: src/App.test.js
Comment (Pradyuman7) [inline src/App.test.js:1]: Is there a reason why this file was removed? I think we should keep it in

File: src/index.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ const root = ReactDOM.createRoot(document.getElementById("root"));
 root.render(
   <React.StrictMode>
     <App />
-  </React.StrictMode>
+  </React.StrictMode>,
```
Comment (Pradyuman7) [inline src/index.js:11]: I think this is not needed

File: src/data.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
+let data = {
```
Comment (Pradyuman7) [inline src/data.js:1]: Why do we have this?

File: src/index.js (hunk 1 above)
Comment (sammychinedu2ky) [reply src/index.js:11]: Placeholder for the python endpoint to be available

File: src/App.test.js
Comment (sammychinedu2ky) [reply src/App.test.js:1]: It was interfering with some npm resolutions. But I can take it back if you want

File: src/data.js (hunk 2 above)
Comment (sammychinedu2ky) [reply src/data.js:1]: Placeholder for the python endpoint to be available

File: src/index.js (hunk 1 above)
Comment (sammychinedu2ky) [reply src/index.js:11]: Prettier formatting added it. Since it's part of the test that is executed

File: src/App.test.js
Comment (Pradyuman7) [reply src/App.test.js:1]: If it's possible to keep it in, please do since it's a test file for App.js

File: src/data.js (hunk 2 above)
Comment (Pradyuman7) [reply src/data.js:1]: Makes sense, can you please add a comment on the data.js file about this so it's easier to understand

**Comment by sammychinedu2ky** [comment]
@Pradyuman7 fixed

**Review by Pradyuman7 — APPROVED** [review]
LGTM 💯

**Pull request merged**